docs/usage/configuration/config_documentation.md

```diff
@@ -462,6 +462,20 @@ See the docs [request log format](../administration/request_log.md).
 * `additional_resources`: Only valid for an 'http' listener. A map of
    additional endpoints which should be loaded via dynamic modules.
 
+Unix socket support (_Added in Synapse 1.87.0_):
+  * `path`: A path and filename for a Unix socket. Make sure it is located in a
+    directory with read and write permissions, and that it already exists (the directory
+    will not be created). Defaults to `None`.
+    * **Note**: The use of both `path` and `port` options for the same `listener` is not
+      compatible.
+    * The `x_forwarded` option defaults to true  when using Unix sockets and can be omitted.
+    * Other options that would not make sense to use with a UNIX socket, such as 
+      `bind_addresses` and `tls` will be ignored and can be removed.
+  * `mode`: The file permissions to set on the UNIX socket. Defaults to `666`
+  * **Note:** Must be set as `type: http` (does not support `metrics` and `manhole`). 
```

Comment:
Doesn't affect the rendered result but it looks like all of these need to be un-indented to match the root list level of the surrounding things. Wish we had some auto-formatting for this.

Reply:
(forgot to link that) Fixed in [85eba85](https://github.com/matrix-org/synapse/pull/15708/commits/85eba85a4eb09496ccf7a2c77848f06e0cfe0178)